tox.ini

```diff
@@ -25,7 +25,9 @@ deps =
 	git+git://github.com/graphite-project/whisper.git#egg=whisper
 	git+git://github.com/graphite-project/ceres.git#egg=ceres
 	pyparsing2: pyparsing
-	django19: Django>=1.9,<1.9.99
+	django19: Django>=1.9,<1.10
+	django110: Django>=1.10,<1.11
+	django111: Django>=1.11,<1.12
```

Comment:
These will allow the `django19` env to pull in pre-release versions of `1.10` etc, we want to keep the same requirement pattern as we had before.